__init__.py

```diff
@@ -75,6 +78,7 @@ def initialize(self):
         if self.examples_enabled:
             self.skill_info_skill = self.settings.get(
                 "examples_skill") or "ovos-skills-info.openvoiceos"
+        self.screenshot_folder = self.settings.get("screenshot_folder", "")
```

Comment:
please read self-settings directly when needed instead of creating a new var, this allows changing path in some UI or selene without a restart of skills service